jupyterlab/pyproject.toml

```diff
@@ -60,5 +60,11 @@ npm = ["jlpm"]
 source_dir = "src"
 build_dir = "qsharp-jupyterlab/labextension"
 
+[tool.hatch.build.targets.wheel]
+packages = ["qsharp-jupyterlab"]
+
+[tool.hatch.build.targets.wheel.force-include]
```

Comment:
Do you want this in the wheel or source distribution?

Reply:
It was in the wheel previously, we don't do an sdist for this but I wanted to make sure the contents were the same as before.